frontend/.env

```
@@ -10,4 +10,4 @@ VITE_GOOGLE_OAUTH_URL="/hack/fakeoauth/oauth2/v2/auth"
 # Codespaces work well if we expose a single port and hosting everything (backend+frontend) there
 # This has to be a full URL not just /hack, because OAuth provider will
 VITE_GOOGLE_REDIRECT_URL="/hack/callback"
-VITE_RAZORPAY_KEY_ID="rzp_test_RTQdscKrA4uMJj"
\ No newline at end of file
+VITE_RAZORPAY_KEY_ID=""
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)